ui/src/app/images/views/ImageList/SyncedImages/SyncedImages.tsx

```diff
@@ -32,9 +33,12 @@ const getImageSyncText = (sources: BootResourceUbuntuSource[]) => {
 
 type Props = {
   formInCard?: boolean;
-};
+} & HTMLProps<HTMLElement>;
 
-const SyncedImages = ({ formInCard = true }: Props): JSX.Element | null => {
+const SyncedImages = ({
+  formInCard = true,
+  ...props
```

Comment:
I think we decided (but never codified) to include the element whose props get spread in the name, i.e. `...stripProps`